package.json

```diff
@@ -14,6 +14,7 @@
     "markdown-it": "^13.0.1",
     "next": "^13.0.6-canary.0",
     "node-sass": "^7.0.3",
+    "prettier": "^2.8.1",
```

Comment:
Let's install prettier globally in local instead of project-wise. It is not required in production.